src/Chapter3.hs

```diff
@@ -970,6 +1078,23 @@ implement the following functions:
 
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
+data DayOfWeek = Mon | Tue | Wed | Thu | Fri | Sat | Sun deriving (Show, Eq, Ord, Enum)
+
+isWeekend :: DayOfWeek -> Bool 
+isWeekend = (>=Sat)
```

Comment:
That is right!
You could also use the instance of `Bounded` in here:

```suggestion
    | day == maxBound = minBound
```

This is also a bit more maintainable. If you decide, for example, to have the "American" system of week days in the data type constructors, you won't need to change this function at all! 🙂